README.md

```diff
@@ -96,17 +92,15 @@ This will generate the following JSON output:
 }
 ```
 
-**Take note of the output. It will be required for the next steps.**
+> [!TIP] Take note of the output. It will be required for the next steps.
 
 Now you can choose whether you would like to use GitHub Actions or Azure DevOps for your deployment.
 
-## 3. a) GitHub Actions
+## 3. a. GitHub Actions
```

Comment:
Please can we have a quick intro paragraph with a links to the sections

For example...

3. Deployment Options

GIthub Actions (with a hyperlink to heading)
Azure DevOps (with a hyperlink to heading)